### PR TITLE
Hide Resource From Sidebar

### PR DESCRIPTION
**Pull request by 0x15f**

### CODE REVIEW

**Comment by 0x15f** [comment]
Can we merge this so I can use the official package with my clients instead of my fork?

File: src/Nova/Comment.php [hunk 1]
```diff
@@ -14,6 +14,8 @@
 
 class Comment extends Resource
 {
+    public static $displayInNavigation = false;
```
Comment (brandonferens) [inline src/Nova/Comment.php]: @0x15f This needs a doc block. Please dupe the one found in Nova Resource.

File: src/Nova/Comment.php [hunk 2]
```diff
@@ -14,6 +14,8 @@
...
+
```
Comment (brandonferens) [inline src/Nova/Comment.php]: Please remove the whitespace on this line as Codacy is failing as a result.

**Comment by brandonferens** [comment]
@0x15f I needed to push out a new version and didn't have time to wait for the above fix updates. I've made reference to you in the change log and have created a new release: `1.0.2`.

**Comment by 0x15f** [comment]
@brandonferens No worries, I'll open another PR that uses a config value. I'm not 100% sure how that will work but I'll figure it out :smile:

**Comment by brandonferens** [comment]
@0x15f Thanks for the suggestion and PR! Feel free to submit more if you see a need.

**Pull request closed without merging**